src/activity/log.rs

```diff
@@ -0,0 +1,46 @@
+use super::spec::*;
+
+use std::net::SocketAddr;
+use std::path::PathBuf;
+use chrono::{DateTime, Utc};
+
+/// Information to log about a process
+#[derive(Serialize,Debug)]
+pub struct ProcessLogInfo {
+    pub name: String,
```

Comment:
Was there a design choice that required making the fields of the struct `pub`? Or a reason you opted to use raw structure construction over a constructor function? E.g. `ProcessInfo { .. }` versus `ProcessInfo::new(..)`

Reply:
I had in the back of my mind the thought that I'd come back and do constructor functions once the design of the types was more solidified; it felt a bit more malleable with just public fields and direct construction. No design-related reason, though.